homeassistant/components/group/light.py

```diff
@@ -159,6 +162,14 @@ def should_poll(self) -> bool:
 
         if ATTR_COLOR_TEMP in kwargs:
             data[ATTR_COLOR_TEMP] = kwargs[ATTR_COLOR_TEMP]
+            for entity_id in list(self._entity_ids):
+                state = self.hass.states.get(entity_id)
+                if state:
+                    support = state.attributes.get(ATTR_SUPPORTED_FEATURES)
+                    # Only pass color temperature to supported entity_id's
+                    if support & SUPPORT_COLOR_TEMP == 0:
```

Comment:
We can only emulate color temp for entities that do support color, so you need to check for that too.

Reply:
Done.